stable/influxdb/templates/secret.yaml

```diff
@@ -1,4 +1,6 @@
 {{- if .Values.setDefaultUser.enabled -}}
+{{- if .Values.setDefaultUser.user.existingSecret -}}
```

Comment:
We know the logic here, If the parameter .`Values.setDefaultUser.user.existingSecret==true`  we do nothing here. But I believe `{{- if  not(.Values.setDefaultUser.user.existingSecret) -}}` may more clearly.